src/pages/TermsOfService.js

```diff
@@ -1,281 +1,341 @@
-import { MarkdownViewer } from 'src/components/markdown'
+import { MarkdownViewer, newWindowLinkRenderer } from 'src/components/markdown'
 import { Ajax } from 'src/libs/ajax'
 import { signOut } from 'src/libs/auth'
 import colors from 'src/libs/colors'
 import { reportError } from 'src/libs/error'
 import { authStore } from 'src/libs/state'
 import * as Style from 'src/libs/style'
 
-// When updating the TOS, make sure you:
-// 1. update the TOS version number in the Ajax call
-// 2. update the dev and prod datastores to have a TOS of that version number
+
+/*
+ * When updating the TOS, make sure you:
+ *   1. update the TOS version number in the Ajax call
+ *   2. update the dev and prod datastores to have a TOS of that version number
+ *
+ * Updating the text here:
+ *   1. If in a google doc, save as .docx
+ *   2. `brew install pandoc`
+ *   3. `pandoc -w markdown -o <out file>.md <in file>.docx`
+ *   4. A bit of replace-all-ing to fix formatting and unescape things will probably be needed
+ *   5. Look out for links that need to be added, emails will become clickable on their own
+ */
+
 const termsOfService = `
-Your access to systems and networks owned by Broad Institute is governed by, and subject to the
-following terms and conditions, as well as all Federal laws, including, but not limited to,
-the Privacy Act, 5 U.S.C. 552a, if the applicable Broad Institute system maintains individual
-Privacy Act information. Your access to Broad Institute systems constitutes your consent to the
-retrieval and disclosure of the information within the scope of your authorized access, subject to
-the Privacy Act, and applicable State and Federal laws.
+Your access to systems and networks owned by The Broad Institute, Inc.
+("Broad") is governed by, and subject to the following terms and
+conditions, as well as all Federal laws, including, but not limited to,
+the Privacy Act, 5 U.S.C. 552a, if the applicable Broad system maintains
+individual Privacy Act information. Your access to Broad systems
+constitutes your consent to the retrieval and disclosure of the
+information within the scope of your authorized access, subject to the
+Privacy Act, and applicable State and Federal laws.
 
 #### Cookies
-Terra uses cookies to enable sign on and other essential features when signed in, and to provide
-statistics to our development team regarding how the site is used. For more information, see our
-privacy policy. Privacy policy, including privacy policy of uploaded data, is [here](/#privacy).
-You agree to read and consider before uploading data.
+
+Terra uses cookies to enable sign on and other essential features when
+signed in, and to provide statistics to our development team regarding
+how the site is used. For more information, see our privacy policy.
+Privacy policy, including privacy policy of uploaded data, is
+[here](/#privacy). You agree to read and consider before uploading data.
```

Comment:
This probably isn't a new problem with this PR, but I just noticed that I can't view the privacy policy. It show briefly and then redisplays the Terms of Service.